docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -635,6 +635,33 @@ The placeholder is displayed when a field is empty:
   ]
 ```
 
+### Categories
+
+You can define a category to group templates in the element template selection list. The category is defined as an object with `id` and `name` properties.
+
+```json
+[
+  {
+    "name": "REST connector",
+    "id": "io.camunda.examples.RestConnector",
+    "description": "A REST API invocation task.",
+    "appliesTo": ["bpmn:ServiceTask"],
+    "category": {
+      "id": "connectors",
+      "name": "Connectors"
+    },
+    "properties": [
+      ...
+    ]
+  }
+]
+```
+
+::note
+The category is optional. If not defined, the template will be displayed in the `Templates` section.
+The ID of the category must be formatted with a kebab-case, e.g., `connectors`, `user-tasks`, etc.
```

Comment:
What do we gain from that statement? What blows up if I define the ID `Foo Bar Yea-100BaWa`? 

Let's only include necessary details, and no more.

Reply:
The IDs generated in the WM were in kebab-case, hence I thought it made sense to continue following that format. However, if we don't have any technical reason behind it, then I'm fine with ditching this constraint. I'll make sure to update the documentation accordingly, while I've already removed the pattern from the schema change.